src/directive.js

```diff
@@ -17,8 +18,11 @@
                  * @returns {}
                  */
                 var validFunc = function(element, validMessage, validation, callback, ctrl) {
-                    var messageElem = element.next(),
-                        messageToShow = validMessage || $validationProvider.getDefaultMsg(validation).success;
+                    var messageElem, node = document.querySelector('[validator-msg-for="' + ctrl.$name + '"]');
```

Comment:
Hi.
In december i don't seen request #70
<https://github.com/huei90/angular-validation/pull/70>
but I need change location for some messages, so I use this attribute for
resolve my problems.

2015-02-11 15:42 GMT+02:00 Huei Tan <notifications@github.com>:

> In src/directive.js
> <https://github.com/huei90/angular-validation/pull/59#discussion_r24494873>
> :
>
> > @@ -17,8 +18,11 @@
> >                   * @returns {}
> >                   */
> >                  var validFunc = function(element, validMessage, validation, callback, ctrl) {
> > -                    var messageElem = element.next(),
> > -                        messageToShow = validMessage || $validationProvider.getDefaultMsg(validation).success;
> > +                    var messageElem, node = document.querySelector('[validator-msg-for="' + ctrl.$name + '"]');
>
> @akozenko <https://github.com/akozenko> May I know why there's the
> attribute validator-msg-for="" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/huei90/angular-validation/pull/59/files#r24494873>.
>